src/frontend/komponenter/Fagsak/Dokumentutsending/DeltBosted/DeltBostedAvtaler.tsx

```diff
@@ -24,7 +24,7 @@ const Container = styled.div`
     margin-left: 3rem;
 `;
 
-export const StyledFamilieDatovelger = styled(FamilieDatovelger)<{ feil: boolean }>`
+export const StyledFamilieDatovelger = styled(FamilieDatovelger)<{ feil?: React.ReactNode }>`
```

Comment:
Dette er for å rette en typefeil. `feil`-propen tar egentlig imot ReactNode som inneholder feilmeldingen. Her i stylingen har vi overstyrt typen til å være en boolean som vi velger farge med. Setter den til å være samme type som i opprinnelig komponent slik at feilmeldingen kan vises